velox/type/tests/TimestampTest.cpp

```diff
@@ -309,6 +309,21 @@ TEST(TimestampTest, outOfRange) {
       t.toTimezone(*timezone), "Timestamp is outside of supported range");
 }
 
+// In debug mode, Timestamp constructor will throw exception if range check
+// fails.
+#ifdef NDEBUG
+TEST(TimestampTest, overflow) {
+  Timestamp t(std::numeric_limits<int64_t>::max(), 0);
+  VELOX_ASSERT_THROW(
```

Comment:
only this test needs to be debug-only, the test that uses allow-overflow doesn't need to be debug-only

It would be nice to verify the result of toMillisAllowOverflow.

Reply:
`toMillisAllowOverflow` is a member of `Timestamp`, and it looks we cannot construct the overflow case with `Timestamp` due to the below check in debug mode: https://github.com/facebookincubator/velox/blob/main/velox/type/Timestamp.h#L96